src/data/icons/svg/WSOHM.svg

```diff
@@ -0,0 +1,8 @@
+<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
```

Comment:
Thanks for the contribution. could you merge master in? we have extra test that do some checks to ensure icons are fine. Thanks

Unfortunately, we can't afford to have `id=`, `class=` or `<style>` in these SVG because we will convert them back into react class.

I still see some style and ids in the SOHM and WSOHM files.